src/libs/actions/IOU.ts

```diff
@@ -1858,23 +1858,31 @@ function buildOnyxDataForInvoice(
     return [optimisticData, successData, failureData];
 }
 
+type BuildOnyxDataForTrackExpenseParams = {
+    chat: {report: OnyxInputValue<OnyxTypes.Report>; previewAction: OnyxInputValue<ReportAction>};
+    iou: {report: OnyxInputValue<OnyxTypes.Report>; createdAction: OptimisticCreatedReportAction; action: OptimisticIOUReportAction};
+    transactionParams: {transaction: OnyxTypes.Transaction; transactionThreadReport: OptimisticChatReport | null; transactionThreadCreatedReportAction: OptimisticCreatedReportAction | null};
+    policyParams: {policy?: OnyxInputValue<OnyxTypes.Policy>; policyTagList?: OnyxInputValue<OnyxTypes.PolicyTagLists>; policyCategories?: OnyxInputValue<OnyxTypes.PolicyCategories>};
```

Comment:
Please remove redundant prefixes. I know the example I linked has some of that too but let's try to remove that going forward. So for example it should be policyParams.tagList vs policyParams.policyTagList. Same for transaction stuff.